recipes/emperor/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "3.8.2" %}
+
+package:
+  name: emperor
+  version: {{ version }}
+
+source:
+  fn: emperor-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/e/emperor/emperor-{{ version }}.tar.gz
+  md5: 2456d7ca823b0bbccf9e235d4ebd8993
+
+build:
+    number: 0
```

Comment:
Please add `skip: true  # [win]` since `scikit-bio` does not exist on Windows ATM.